app/types/transport/responses/task/change-task-status.ts

```diff
@@ -0,0 +1,28 @@
+import { ResponseMessage } from 'ctproto';
+import { Status } from 'types/entities';
+import Task from '../../../entities/task';
+
+/**
+ * Response for 'change-task-status' message
+ */
+export interface ChangeTaskStatusResponsePayload {
+  /**
+   * Updated task data
+   */
+  task?: Task | null
+
+  /**
+   * Updated previous status object (if not moved from unsorted column)
```

Comment:
please describe for what reason we return it? The client already knows about the previous status before the request isn't it?

Reply:
yep, it's not needed, removed